Dockerfile.acceptance

```diff
@@ -10,10 +15,12 @@ COPY ./ .
 RUN make build-test
 
 FROM scratch
-WORKDIR /etc/deviceconfig
 EXPOSE 8080
-COPY ./config.yaml .
-COPY --from=builder /go/src/github.com/mendersoftware/deviceconfig/bin/deviceconfig.test \
+COPY --from=builder /etc_extra/ /etc/
```

Comment:
before that `COPY` is run, there is no `nobody` to associate this with in the image, but with the `nobody` `chown` in the `builder` phase this is carried over

Reply:
yeah ok, but it realates to the other question that Alf also raised in the other PR, about the owner ship. technically you could chown a bit later. I am ok with that, I only feel that it works a bit by accident. the other concerns I raised in the other thread.

Reply:
If you have an idea on how to chown later in all ears, but there is no nobody user or group in the scratch based step and also no chown command, so the only way I know it's based on the --chown copy

Reply:
I am not blocking this, since it is approved, just being curious, a couple of lines below you happily use --chown so why cant we use it here? and if you consider it chicken-egg problem (the presence of files in /etc vs the existence of the user based on those files) how is it possible that group, passwd and shadow files retain nobody:nobody?

Reply:
No worries I also want to get this classified as much as possible, since I also might misunderstand things... So yes this is a chicken egg problem that the --chown would not be able to use the user/group info unless the files are present... Doing an additional copy might be possible but would add an additional layer to the image. I really encourage you to play around with one of the dockerfiles + the dive tool and see what you run into...

Reply:
I do like fun and games, especially when you organise them :> that is: I am playing with it, as much as time permits. so far I could not break it, today I will try in the evening to tackle the chown thing a bit.